src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -6,6 +6,7 @@ import { KibanaMapLayer } from 'ui/vis/map/kibana_map_layer';
 import { truncatedColorMaps } from 'ui/vislib/components/color/truncated_colormaps';
 import * as topojson from 'topojson-client';
 import { toastNotifications } from 'ui/notify';
+import * as colorUtil from 'ui/vis/map/color_util';
```

Comment:
Could we - in a separate PR - maybe move the `map` folder out of the `vis` folder? I would prefer in the long run, if we could remove anything from that folder, that is not core vis model classes (like vis and vis types) and classes that are used in the overall rendering pipeline (like request and response handlers). It's not urgent at all, but maybe we could keep this in the back of our minds, once a feature freeze is not close :)